cmd/access/node_builder/access_node_builder.go

```diff
@@ -993,6 +997,8 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 					return nil, err
 				}
 
+				builder.stopControl.RegisterHeightRecorder(builder.ExecutionIndexer)
```

Comment:
if we're using `stopControl` here, we'll need to add it as a dependency to `builder.IndexerDependencies`, otherwise node bootstrapping could panic if it's not created yet. same comment for the observer version